roles/ara_api/tasks/install/podman.yaml

```diff
@@ -0,0 +1,59 @@
+---
+- name: Detecting existing PyPI installation
+  ansible.builtin.stat:
+    path: "{{ ara_api_venv_path }}"
+  register: existing_pypi_install
+
+- name: Remove PyPI virtualenv
+  ansible.builtin.file:
+    path: "{{ ara_api_venv_path }}"
+    state: absent
+  when: existing_pypi_install['stat']['exists']
+
+- name: Detecting existing sqlite database
+  ansible.builtin.stat:
+    path: "{{ ara_api_database_name }}"
+  register: existing_sqlite_database
+
+- name: Move sqlite database to new location
+  ansible.builtin.command: mv {{ ara_api_database_name }} {{ ara_api_root_dir }}/ansible.sqlite
+  when:
+    - existing_sqlite_database['stat']['exists']
+    - ara_api_database_engine == 'django.db.backends.sqlite3'
```

Comment:
I think I understand the intent here but I'm not sure whether we should do this automatically.

The collection doesn't (yet) do anything special when switching between installation types or database backends so it feels a bit out of place to go out and delete venvs and move files around.

If the concern is that there might already be an existing installation, maybe we could print a helpful debug message about it and fail the playbook such that the user is given the opportunity to clean it up or automate it outside the collection.

We can always reconsider if it turns out to be a big issue or something we really want to support.

Reply:
I put it behind a flag for now :-) and I added notice in the script to notify the user.